src/backend/web/templates/event_partials/event_matchstat_table.html

```diff
@@ -0,0 +1,67 @@
+{% if matchstat_choices|length > 0 %}
+<h3><abbr title="Component Offensive Power Ratings">COPRs</abbr> - Top 15 (<a href="/opr">?</a>)</h3>
+<div class="btn-group">
+  <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+          aria-expanded="false">
+    <span id="coprSelector">Choose component</span> <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu">
+    {% for component, component_id in matchstat_dropdown_id_map.items() %}
+        <li>
+            <a href="#" id="dropdown_{{ component_id }}">{{ component }}</a>
+        </li>
+    {% endfor %}
+  </ul>
+</div>
+
+<table class="table table-condensed table-striped table-center">
+    <thead>
+        <tr>
+            <th>Team</th>
+            <th id="componentColHeader"><abbr title="Component Offensive Power Rating">COPR</abbr></th>
+        </tr>
+    </thead>
+    <tbody id="coprTableBody">
+    </tbody>
+</table>
+
+<script>
+    const coprs = JSON.parse('{{ matchstat_json|safe }}');
+
+    function buildCoprTable(component) {
+        document.getElementById('componentColHeader').innerHTML = `COPR`;
+        document.getElementById('coprSelector').innerText = component;
+        document.getElementById('coprTableBody').innerHTML = '';
+        coprs[component].forEach((e) => {
+            const teamId = e[0].slice(3);
+            const teamLink = document.createElement('a');
+            teamLink.href = `/team/${teamId}/{{ event.year }}`;
+            teamLink.innerText = teamId;
+            const teamTd = document.createElement('td');
+            teamTd.appendChild(teamLink);
+
+            const coprTd = document.createElement('td');
+            coprTd.innerText = e[1].toFixed(2); // round to 2 decimal places
+
+            const tr = document.createElement('tr');
+            tr.appendChild(teamTd);
+            tr.appendChild(coprTd);
+            document.getElementById('coprTableBody').appendChild(tr);
+        });
+    }

```

Comment:
Why do we need to do this in Javascript? Can we really not do this with Jinja?

Reply:
Truthfully, this is just how I've done it on my own projects in the past. I'm not sure how I would do it with templating. I think this is the easiest approach, but I am open to other methods of doing it if you have ideas towards it.

Reply:
Lemme take a pass at this later. We can always merge this as-is (I'm not opposed to this approach - I'd just rather do it in templating)